在D:\Programming\Python\Anaconda3\envs\py37\Lib\site-packages\django\views中的debug.py

設定成了utf-8
def get_traceback_html(self):
       """Return HTML version of debug 500 HTTP error page."""
       # with Path(CURRENT_DIR, 'templates', 'technical_500.html').open() as fh:
       with Path(CURRENT_DIR, 'templates', 'technical_500.html').open(encoding='utf-8') as fh:
           t = DEBUG_ENGINE.from_string(fh.read())
       c = Context(self.get_traceback_data(), use_l10n=False)
       return t.render(c)